marimo/_messaging/ops.py

```diff
@@ -192,6 +192,16 @@ def broadcast_error(
         ).broadcast()
 
 
+@dataclass
+class FunctionCallResult(Op):
+    """Result of calling a function."""
+
+    function_call_id: str
+    return_value: JSONType
```

Comment:
i wonder if we should allow returning a structured error. something that is human/UI friendly:
```
{
  title: "",
  message: "",
}
```

and up to the component to figured out how it wants to display it